src/model/document.ts

```diff
@@ -335,7 +335,8 @@ export class Document implements Feature, Queryable {
       }
       if (feature.kinds.has('import') && options.imported) {
         const imprt = feature as Import;
-        if (options.externalPackages || !Package.isExternal(imprt.url)) {
+        if (options.externalPackages ||
```

Comment:
can you add parens here? They make it so much easier to read/maintain when there are multiple logical operators in a conditional.